portfolio/src/main/webapp/js/activities.js

```diff
@@ -0,0 +1,96 @@
+let scroller = null;
+/**
+ * Gathers repos from GitHub and scrolls through them in the About Me section.
+ */
+async function startActivitiesScroll() {
+  let repos = await fetch("https://api.github.com/users/Richie78321/repos")
+    .then(res => res.json());
+  repos =
+    repos.filter(repo => repo.description && repo.description.length !== 0);
+
+  scroller = new Scroller(repos);
+  scroller.start();
+}
+
+/**
+ * Scrolls through a set of GitHub repository descriptions at a set speed.
+ */
+class Scroller {
+  /**
+   * @param {Array<Object>} repos 
+   */
+  constructor(repos) {
+    /**
+     * Collection of user GitHub repos.
```

Comment:
Maybe add a link to the docs where these repo objects are described?

Reply:
Solid point. I'll add that in.